01-introduction.Rmd

```diff
@@ -2,34 +2,33 @@
-
+This chapter is a welcome to readers and an overview of the book. It describes challenges with learning data science in education, including the complexity of the data scientist role. Finally, this chapter introduces data analysis walkthroughs as a way to learn data science in the education field.
 
 Dear Data Scientists, Educators, and Data Scientists who are Educators: 
 
-This book is a warm welcome and an invitation to learn with a great community. If you're a data scientist in education or an educator in data science, your role probably isn't straightforward. This book is our contribution to a growing movement to merge the paths of data analysis and education. We wrote this book to make your first step on that path a clearer and a less scary. 
+This book is a warm welcome and an invitation to learn with a great community. If you're a data scientist in education or an educator in data science, your role probably isn't straightforward. This book is our contribution to a growing movement to merge the paths of data analysis and education. We wrote this book to make your first step on that path clearer and less scary.
 
-Whether you're a data scientist using your skills in an education job or an educator who wants to learn data science skills, we invite you to read this book and put these techniques to work in the real world. Your work in the education community will help decide how education and data science come together going forward.
+Whether you're a data scientist working in education or an educator wanting to learn data science, we invite you to read this book and put these techniques to work in the real world. Your work in the education community will help decide how education and data science come together.
 
 ## Learning data science in education
 
 Over the coming chapters, you'll explore examples of what data science in education looks like. But first, let's talk about why data science in education is not easily definable. 
 
-Being a data scientist in education is challenging because there is not a universal vision for that role yet. For starters, education as a field is complex. If education were a building, it would be multi-storied with many rooms. There are privately and publicly funded schools. There are more than 18 possible grade levels. Students can learn alone or with others in a classroom. Students start school, leave school, finish school, and return to school. 
+Being a data scientist in education is challenging because there is not a universal vision for what that means. For starters, education as a field is complex. If education were a building, it would be multi-storied with many rooms. There are privately and publicly funded schools. There are more than 18 possible grade levels. Students can learn alone or with others in a classroom. Students start school, leave school, finish school, and return to school. 
```

Comment:
```suggestion
Being a data scientist in education is challenging because there is not a universal vision of what that means. For starters, education as a field is complex. If education were a building, it would be multi-storied with many rooms. There are privately and publicly funded schools. There are more than 18 possible grade levels. Students can learn alone or with others in a classroom. Students start school, leave school, finish school, and return to school. 
```